sorc/ncep_post.fd/WETFRZLVL.f

```diff
@@ -72,8 +72,8 @@ SUBROUTINE WETFRZLVL(TWET,ZWET)
 !!$omp  parallel do
 !!$omp& private(delt,delz,htsfc,l,llmh
 !!$omp&         tsfc,zl,zu)
-      DO 20 J=JSTA,JEND
```

Comment:
Can you give this loop a name? such as loopj and loops?

Reply:
@hu5970, from the original block, the "goto" can be either I or J. In the new changes, the labels "20" are removed from I and J loops. So cycle or exit can be either I or J. Same as the original one.